pylitterbot/robot/litterrobot4.py

```diff
@@ -224,6 +228,8 @@ def firmware_update_triggered(self) -> bool:
     @property
     def hopper_status(self) -> HopperStatus | None:
         """Return the hopper status."""
+        if self._data.get("hopperStatus") is None:
+            return None
```

Comment:
The `to_enum` method already handles returning None if the value passed to it is None